speed-dreams/source-2.2.3/src/libs/tgfdata/tracks.cpp

```diff
@@ -308,6 +308,9 @@ GfTrack* GfTracks::getFirstUsableTrack(const std::string& strCatId,
 									   const std::string& strFromTrackId,
 									   int nSearchDir, bool bSkipFrom) const
 {
+    // SIMULATED DRIVING ASSISTANCE change: Skip the test road so that it won't show in the track selector
+    if (strCatId == "road") return 0;
```

Comment:
This is quite hacky, instead just remove the test_highway

Reply:
Yeah i tried but the upgrade test don't work. And i can't fix that before friday